### PR TITLE
[TIDAL-2039] Email selector menu can be the same width as input

### PR DESCRIPTION
**Pull request by BeirlaenAaron**
### Description

[Jira](https://teamleader.atlassian.net/browse/TIDAL-2039)

This PR allows the autocomplete menu of the email selector to take the same width as it's input
(Also makes the space between the input and the menu a little bit smaller as requested in the ticket)

#### Screenshot before this PR

image

#### Screenshot after this PR

If the `menuFullWidth` prop is on the email selector, it should look like this:
image

### Breaking changes

N/A
It's set as an optional prop to avoid breaking changes

### CODE REVIEW

**Review by stefaandevylder — CHANGES_REQUESTED** [review]
This needs to be added to the changelog

**Review by driesd — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +19,7 @@
 
 ## [14.6.1] - 2022-06-10
 
-### Fixed
+### Fixed=======
```
Comment (lowiebenoot) [inline CHANGELOG.md:22]: I think you did a little oopsie here.

**Comment by lowiebenoot** [comment]
In which case would it not look good? I would also opt for always giving it the full width, just like our Select component does.

File: CHANGELOG.md (hunk 1 above)
Comment (BeirlaenAaron) [reply CHANGELOG.md]: That is indeed an oopsie 😅

**Review by qubis741 — APPROVED** [review]

**Review by lorgan3 — APPROVED** [review]

**Review by jelledc — APPROVED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Comment by stefaandevylder** [comment]
MC'ing so UI can be deployed

**Pull request merged**